<ui version="4.0" >
 <author>Alexander Saal - alex.saal@gmx.de</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DBoxFE_ProfileWizard</class>
 <widget class="QDialog" name="DBoxFE_ProfileWizard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>731</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>731</width>
    <height>391</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Gamewizard</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/dboxfe.qrc" >:/pics/images/default.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>340</y>
     <width>711</width>
     <height>18</height>
    </rect>
   </property>
   <property name="flat" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>350</y>
     <width>711</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnHelp" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="minimumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;Help</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>166</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="minimumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnNext" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="minimumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text" >
       <string>&amp;Next</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>165</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnAbort" >
      <property name="minimumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>110</width>
        <height>26</height>
       </size>
      </property>
      <property name="text" >
       <string>&amp;Abort</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="pageSearchGame" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>10</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>213</y>
      <width>531</width>
      <height>102</height>
     </rect>
    </property>
    <property name="title" >
     <string>Directory</string>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="LEDirectory" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnSearch" >
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Search games</string>
           </property>
           <property name="whatsThis" >
            <string>Search games</string>
           </property>
           <property name="icon" >
            <iconset resource="../res/dboxfe.qrc" >:/pics/images/misc.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectDir" >
           <property name="minimumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Open directory</string>
           </property>
           <property name="whatsThis" >
            <string>Open directory</string>
           </property>
           <property name="icon" >
            <iconset resource="../res/dboxfe.qrc" >:/pics/images/folder-open_16.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gBoxGames" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title" >
     <string>Games found</string>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="chkBoxAddGameToAutexec" >
       <property name="text" >
        <string>add Game/s to autexec section</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QCheckBox" name="chkBoxSelectAll" >
       <property name="text" >
        <string>select all Games</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" >
      <widget class="QTreeWidget" name="lwGames" >
       <property name="selectionMode" >
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <column>
        <property name="text" >
         <string>Game</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Directory</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="pageCreateProfiles" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>410</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="gBoxFinish" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title" >
     <string>Finish</string>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>9</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="lblSaveCfg" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="pixmap" >
            <pixmap resource="../res/dboxfe.qrc" >:/pics/images/arrow-forward_16.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCreateGP" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="pixmap" >
            <pixmap resource="../res/dboxfe.qrc" >:/pics/images/arrow-forward_16.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFinish" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="pixmap" >
            <pixmap resource="../res/dboxfe.qrc" >:/pics/images/arrow-forward_16.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Save configuration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Create game profiles</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Finish</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="lblPicture" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>168</width>
     <height>318</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap resource="../res/dboxfe.qrc" >:/pics/images/wiz_internet.png</pixmap>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>LEDirectory</tabstop>
  <tabstop>btnSelectDir</tabstop>
  <tabstop>lwGames</tabstop>
  <tabstop>chkBoxSelectAll</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>btnNext</tabstop>
  <tabstop>btnBack</tabstop>
  <tabstop>btnAbort</tabstop>
  <tabstop>btnHelp</tabstop>
 </tabstops>
 <resources>
  <include location="../res/dboxfe.qrc" />
 </resources>
 <connections/>
</ui>
